src/NServiceBus.RavenDB/Internal/RavenDBVersionValidator.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace NServiceBus.Persistence.RavenDB;
+
+using System;
+using NuGet.Versioning;
+
+static class MinimumRequiredRavenDbServerVersion
+{
+    static readonly string Version = "5.2.0";
+
+    public static void Validate(string ravenDbVersionString, string minimumRequiredVersion)
+    {
+        var minimumRavenDbVersion = new NuGetVersion(minimumRequiredVersion);
```

Comment:
Out or curiosity what does the NuGetVersion give compared to using https://learn.microsoft.com/en-us/dotnet/api/system.version.-ctor?view=net-8.0 ?